server/api/api.go

```diff
@@ -720,6 +719,7 @@ func filterOrphanBlocks(blocks []model.Block) (ret []model.Block) {
 	return blocks
 }
 
+/*
```

Comment:
I'm not a big fan of commenting things, if it is no longer needed, I guess we can delete it.